### PR TITLE
Simplify Dockerfile and Gitbook structure

### PR DESCRIPTION
**Pull request by xpicox**
In this PR:
- Update instructions on `README.md`
- Changes to `Dockerfile`:
  - Use `node:alpine` as the base image
  - Remove unnecessary instructions
- Changes to `web/`:
  - Add `book.json` and `styles/website.css` (in view of a fix for `all.Dockerfile` on `master`)
  - Fix navigation buttons
  - Add missing file `versions.json`

### CODE REVIEW

**Comment by fmontesi** [comment]
@bmaschio @xiroV can you have a look?

File: Dockerfile [hunk 1]
```diff
@@ -1,28 +1,19 @@
...
-# sed append line after match
-RUN sed -i '/^{/a \    "root": "./web",' book.json
-RUN npm update
+FROM node:alpine
 RUN npm install -g gitbook-cli
-RUN npm install mv
-RUN npm install gitbook-plugin-highlight-jolie
-RUN npm install gitbook-plugin-theme-jolie
-RUN npm install gitbook-plugin-logo
-RUN npm install gitbook-plugin-collapsible-chapters
```
Comment (xiroV) [inline Dockerfile:24]: By removing these plugins we don't have the syntax-highlighting, theme or logo anymore. Did you move these somewhere else?

File: README.md [hunk 2]
```diff
@@ -1,17 +1,52 @@
...
 ```
-docker build -t jolie-docs . 
+docker-compose build
 ```
 
 Then run it with
 
 ```
-docker run --rm -it -p 8080:8080 -d jolie-docs
+docker-compose up -d
```
Comment (xiroV) [inline README.md:28]: I installed docker-compose, but I was unable to get this workflow working. It says `Starting docs_jolie-docs_1 ... done` upon running, but I cannot access the docs from the browser.

**Review by xiroV — COMMENTED** [review]
I'm having some problems getting the docker-compose workflow to work. Maybe I misunderstood something. Also, it looks like you removed some of the plugins we are using, and I haven't been able to find if you simply moved the installation of these somewhere else, or not.

File: Dockerfile (hunk 1 above)
Comment (xpicox) [reply Dockerfile:24]: Those plugins are automatically installed by `gitbook install`.

File: README.md (hunk 2 above)
Comment (xpicox) [reply README.md:28]: I think the problem is that the book takes a while to build. Just wait a few minutes and then try to access the docs. Also try to run `docker-compose up` without the option `-d` to get some feedback from the running image.

Comment (xiroV) [reply README.md:28]: Thanks, I must have been too quick.

**Review by xiroV — APPROVED** [review]
After the clarification from @xpicox I got it working, and I think it looks good.

**Pull request merged**